src/index.html

```diff
@@ -1,14 +1,59 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="index.html">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav-links">
+          <li class="nav-link">
+            <a href="#Apple" class="nav-link is-active">Apple</a>
+          </li>
+          <li class="nav-link">
+            <a href="#Samsung" class="nav-link">Samsung</a>
+          </li>
```

Comment:
Don't forget about code style, you can check it in (checkList)[https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md]
![image](https://github.com/mate-academy/layout_moyo-header/assets/91411692/6110dc68-9468-470a-9ea3-3ea44ddca979)

```suggestion
      </a>
      
      <nav class="nav">
        <ul class="nav-links">
          <li class="nav-link">
            <a href="#Apple" class="nav-link is-active">Apple</a>
          </li>
          
          <li class="nav-link">
            <a href="#Samsung" class="nav-link">Samsung</a>
          </li>
```